### ISSUE TITLE
Excluding the property drawer from template expansion

### ISSUE THREAD

**Issue by akirak** [Kungsgeten/yankpad#26]
My org-capture setting automatically adds an ID to each captured entry. When I create a new snippet using `yankpad-capture-snippet` command, an ID is assigned to the snippet as well, and whenever I expand the snippet using `yankpad-insert`, a property drawer containing the ID of the snippet is prepended to the result. I don't want this to happen. 

Could you add support for preventing the property drawer of a template from being inserted as part of a result?

**Comment by akirak** [comment]
The issue has been fixed, and here is how it went wrong:

After following your instruction, I found that function included in `org-capture-finalize-hook`, but [this condition test](https://github.com/Kungsgeten/yankpad/blob/master/yankpad.el#L287) seems to have failed. `org-capture-last-stored-marker` contains an absolute path, while my `yankpad-file` is abbreviated, i.e. beginning with `~`, so the test failed. After setting `yankpad-file` in an absolute path using `expand-file-name`, a created snippet no longer has an ID. Thank you for your response.
